OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/UserManager.kt

```diff
@@ -219,4 +235,30 @@ internal open class UserManager(
     override fun getTags(): Map<String, String> {
         return _propertiesModel.tags.toMap()
     }
+
+    override fun addObserver(observer: IUserStateObserver) {
+        changeHandlersNotifier.subscribe(observer)
+    }
+
+    override fun removeObserver(observer: IUserStateObserver) {
+        changeHandlersNotifier.unsubscribe(observer)
+    }
+
+    override fun onModelReplaced(
+        model: IdentityModel,
+        tag: String,
+    ) { }
+
+    override fun onModelUpdated(
+        args: ModelChangedArgs,
+        tag: String,
+    ) {
+        if (args.property == IdentityConstants.ONESIGNAL_ID) {
+            val newUserState = UserState(args.newValue.toString(), externalId)
+            this.changeHandlersNotifier.fire {
+                it.onUserStateChange(UserChangedState(newUserState))
+            }
+        }
+        Logging.debug(args.property)
```

Comment:
I think this log prints just "onesignal_id" and was left over from testing? Let's remove it. I don't think it's necessary to replace with an informative log here either.